source/codegen/metadata/nidigital/CHANGES.md

```diff
@@ -0,0 +1,8 @@
+# Changes in metadata from the nimi-python metadata
+
+# Changes in metadata from the base metadata
+
+## functions.py
+
+The following function was added :
+- WriteStatic
```

Comment:
Currently the target branch does not have changes.md, so I have added in this PR, will take care of it while merging.